internal/adapters/cloudformation/aws/rds/instance.go

```diff
@@ -25,9 +25,10 @@ func getClustersAndInstances(ctx parser.FileContext) (clusters []rds.Cluster, or
 				EncryptStorage: r.GetBoolProperty("StorageEncrypted"),
 				KMSKeyID:       r.GetStringProperty("KmsKeyId"),
 			},
-			PublicAccess:   r.GetBoolProperty("PubliclyAccessible", true),
-			Engine:         r.GetStringProperty("Engine"),
-			IAMAuthEnabled: r.GetBoolProperty("EnableIAMDatabaseAuthentication"),
+			PublicAccess:       r.GetBoolProperty("PubliclyAccessible", true),
+			Engine:             r.GetStringProperty("Engine"),
+			IAMAuthEnabled:     r.GetBoolProperty("EnableIAMDatabaseAuthentication"),
+			DeletionProtection: r.GetBoolProperty("DeletionProtection", false),
```

Comment:
Here we are adding the new `DeletionProtection` property to the CloudFormation adapter - this is pulling the status of Deletion Protection for each DB instance defined in CloudFormation yaml.